pkg/datarepair/repairer/repairer.go

```diff
@@ -150,5 +153,16 @@ func (service *Service) worker(ctx context.Context, seg *pb.InjuredSegment) (err
 	if err != nil {
 		return Error.New("repair delete failed: %v", err)
 	}
+
+	insertedTime := seg.GetInsertedTime()
+	// do not send metrics if segment was added before the InsertedTime field was added
```

Comment:
why not?

Reply:
we would end up sending a bunch of inaccurate times to datascience about how long segments are repaired. Specifically for segments added before this PR. They would have the 0 time which would throw off `time_since_checker_queue`